CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Version 1.2.2
+
+- Fix: If model is empty, after submit model = product name
```

Comment:
Our listings team is going to suspend listings that do this too often. We should require model.

Reply:
Aille,  can you explain it to Aabesh, it's his request ?
Thanks you!

Reply:
If the model is empty, just do not send it in the request. On our end, we will try to guess the correct model from the title and the rest of the data you send.

Reply:
Ok Kyle I will fix it.

thank you